components/style/values/generics/svg.rs

```diff
@@ -101,14 +102,44 @@ impl<ColorType: Parse, UrlPaintServer: Parse> Parse for SVGPaint<ColorType, UrlP
 /// https://www.w3.org/TR/SVG11/painting.html#StrokeProperties
 #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 #[derive(Clone, Copy, Debug, PartialEq, ToCss, HasViewportPercentage)]
-#[derive(ToComputedValue, ToAnimatedValue, ComputeSquaredDistance)]
+#[derive(ToComputedValue, ToAnimatedValue)]
 pub enum SvgLengthOrPercentageOrNumber<LengthOrPercentageType, NumberType> {
     /// <length> | <percentage>
     LengthOrPercentage(LengthOrPercentageType),
     /// <number>
     Number(NumberType),
 }
 
+impl<L, N> ComputeSquaredDistance for SvgLengthOrPercentageOrNumber<L, N>
+    where NumberOrPercentage: From<L> + From<N> + ComputeSquaredDistance,
+          L: ComputeSquaredDistance + Copy,
+          N: ComputeSquaredDistance + Copy
+{
+    #[inline]
+    fn compute_squared_distance(&self, other: &Self) -> Result<SquaredDistance, ()> {
+        match (self, other) {
+            (&SvgLengthOrPercentageOrNumber::LengthOrPercentage(ref from),
+             &SvgLengthOrPercentageOrNumber::LengthOrPercentage(ref to)) =>
```

Comment:
Visual indenting is deprecated, and match arm bodies going on their own line should be braced.

```rust
(
    &SvgLengthOrPercentageOrNumber::LengthOrPercentage(ref from),
    &SvgLengthOrPercentageOrNumber::LengthOrPercentage(ref to),
) => {
    from.compute_squared_distance(to)
},
```